discord/client.py

```diff
@@ -250,6 +267,15 @@ def _get_state(self, **options: Any) -> ConnectionState:
 
     def _handle_ready(self) -> None:
         self._ready.set()
+        
+    def __str__(self):
+        return self._connection.user.name
```

Comment:
`self.user` also works

Reply:
a little more uncomplicated
```suggestion
        return self.user.name
```